sql-delete/puritanical.sql

```diff
@@ -0,0 +1,6 @@
+DELETE
+  from
+    "films"
+  WHERE
+    "rating" = 'G'
+  returning *;
```

Comment:
Delete films where the rating is _not_ G:

---

![image](https://user-images.githubusercontent.com/16143771/197241910-0470f423-e5b6-44b2-b9c8-0d0697101b56.png)

Reply:
<img width="1150" alt="Screen Shot 2022-10-21 at 9 36 49 AM" src="https://user-images.githubusercontent.com/103021766/197245865-93ea5a0c-6ecc-4017-a748-8bf2433ffa34.png">